<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventWidget</class>
 <widget class="QWidget" name="EventWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>346</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Event Widget</string>
  </property>
  <layout class="QVBoxLayout" name="eventModeMainLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="noSlicingGroup">
     <property name="title">
      <string>No Slicing</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="disabledSlicingButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Disable Slicing</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">slicingButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="uniformGroup">
     <property name="title">
      <string>Uniform slicing</string>
     </property>
     <layout class="QVBoxLayout" name="uniformLayoutContainer">
      <item>
       <layout class="QGridLayout" name="uniformLayout0">
        <item row="0" column="0">
         <widget class="QRadioButton" name="uniformEvenButton">
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">slicingButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="uniformEvenLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>even time slices</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="uniformEvenSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="uniformButton">
          <attribute name="buttonGroup">
           <string notr="true">slicingButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="uniformLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>(sec) slices</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <spacer name="uniformSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="uniformEvenEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="uniformEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>100000000000.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="customGroup">
     <property name="title">
      <string>Custom slicing</string>
     </property>
     <layout class="QHBoxLayout" name="customLayout">
      <item>
       <widget class="QRadioButton" name="customButton">
        <attribute name="buttonGroup">
         <string notr="true">slicingButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="customLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Python list (sec)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="customEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>A comma separated list of values indicating the times (in seconds) at which to slice the data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="customSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="logValueGroup">
     <property name="title">
      <string>Slicing by log value</string>
     </property>
     <layout class="QHBoxLayout" name="logValueLayout">
      <item>
       <widget class="QRadioButton" name="logValueButton">
        <attribute name="buttonGroup">
         <string notr="true">slicingButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="logValueLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Python list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="logValueEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>A comma separated list of values indicating the log values at which to slice the data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="logValueTypeLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Log name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="logValueTypeEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>The name of the log value to slice on</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="logValueSpacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="slicingButtonGroup"/>
 </buttongroups>
</ui>
